tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+  </stdio>
+  <command><![CDATA[
+  sistr 
+    -f $output_format
+    #if $output_format == "csv"
+      -o sistr-report.csv
+    #elif $output_format == "json"
+      -o sistr-report.json
+    #end if
+    -p $cgmlst_profiles
+    -n $novel_alleles
+    -a $alleles_output
+    -T $tmp_dir
+    #if $keep_tmp
+      -K
+    #end if
+    #if $use_full_cgmlst_db
+      --use-full-cgmlst-db
+    #end if
+    #if $no_cgmlst
+      --no-cgmlst
+    #end if
+    #if $run_mash
+      -m
+    #end if
+    #if $qc
+      --qc
+    #end if
+    #if $threads > 1
+      -t $threads
+    #end if
+    #if $verbosity == 1
+      -v
+    #elif $verbosity == 2
+      -vv
+    #elif $verbosity == 3
+      -vvv
+    #end if
+    ${" ".join(map(str, $input_fastas))}
+  ]]></command>
+  <inputs>
+    <param 
+      name="input_fastas" 
+      type="data" 
+      label="Input Genome(s)" 
+      optional="false" 
+      multiple="true"
+      format="fasta"
+      />
+    <param 
+      name="output_format" 
+      type="select" 
+      label="Results output format"
+      multiple="false" >
+      <option value="csv" selected="true">
+        CSV (Comma Separated Values)
+      </option>
+      <option value="json">
+        JSON (JavaScript Object Notation)
+      </option>
+    </param>
+    <param 
+      name="use_full_cgmlst_db"
+      type="boolean"
+      checked="false"
+      label="Use full cgMLST database for serovar prediction. About 10X slower with equivalent results to reduced centroid allele database."
+      />
+    <param 
+      name="run_mash"
+      type="boolean"
+      checked="true"
+      label="Run Mash MinHash-based serovar prediction"
+      />
+    <param 
+      name="no_cgmlst"
+      type="boolean"
+      checked="false"
+      label="Skip running cgMLST-based serovar prediction"
+      />
+    <param 
+      name="qc"
+      type="boolean"
+      checked="true"
+      label="Basic QC of results"
+      />
+    <param
+      name="tmp_dir"
+      type="text"
+      value="/tmp"
+      label="Temporary analysis directory"
+     />
+    <param 
+      name="keep_tmp"
+      type="boolean"
+      checked="false"
+      label="Keep temporary analysis directory"
+      />
+    <param 
+      name="threads"
+      type="integer"
+      value="1"
+      label="Number of threads (1 genome will only use 1 thread)"
+      />
+    <param 
+      name="verbosity"
+      type="select" 
+      label="Logging verbosity">
+      <option value="0">
+        Error messages only
+      </option>
+      <option value="1">
+        Show warning messages
+      </option>
+      <option value="2" selected="true">
+        Show info messages
+      </option>
+      <option value="3">
+        Show debug messages
+      </option>
+    </param>
+  </inputs>
+  <outputs>
+    <!-- TODO: Fix genome fasta name mangling by Galaxy; try to use original name if possible -->
+    <data 
+      name="output_prediction_csv" 
+      format="tabular" 
```

Comment:
I've added the option to output to ``tab``/tab-delimited format in v0.3.3. This output format is the default for the Galaxy tool.